node/cmd/guardiand/node.go

```diff
@@ -561,35 +567,35 @@ func runNode(cmd *cobra.Command, args []string) {
 		logger.Fatal("Please specify --nodeName")
 	}
 
-	if *solanaContract == "" {
+	if *solanaContract == "" && !*unsafeDevMode {
 		logger.Fatal("Please specify --solanaContract")
 	}
-	if *solanaWsRPC == "" {
+	if *solanaWsRPC == "" && !*unsafeDevMode {
 		logger.Fatal("Please specify --solanaWsUrl")
 	}
-	if *solanaRPC == "" {
+	if *solanaRPC == "" && !*unsafeDevMode {
 		logger.Fatal("Please specify --solanaUrl")
 	}
 
-	if *terraWS == "" {
+	if *terraWS == "" && !*unsafeDevMode {
 		logger.Fatal("Please specify --terraWS")
 	}
-	if *terraLCD == "" {
+	if *terraLCD == "" && !*unsafeDevMode {
 		logger.Fatal("Please specify --terraLCD")
 	}
-	if *terraContract == "" {
+	if *terraContract == "" && !*unsafeDevMode {
 		logger.Fatal("Please specify --terraContract")
 	}
-	if *terra2WS == "" {
+	if *terra2WS == "" && !*unsafeDevMode {
 		logger.Fatal("Please specify --terra2WS")
 	}
-	if *terra2LCD == "" {
+	if *terra2LCD == "" && !*unsafeDevMode {
 		logger.Fatal("Please specify --terra2LCD")
 	}
-	if *terra2Contract == "" {
+	if *terra2Contract == "" && !*unsafeDevMode {
 		logger.Fatal("Please specify --terra2Contract")
 	}
-	if *testnetMode || *unsafeDevMode {
+	if *testnetMode {
 		if *algorandIndexerRPC == "" {
 			logger.Fatal("Please specify --algorandIndexerRPC")
 		}
```

Comment:
I'd be tempted do simply wrap all of this logic in a single:

```
if *unsaeDevMode {
   ... logic here ...
}
```

As it would be a bit cleaner and less to update. When someone adds the next optional chain it's harder to break it if they put this logic inside a single conditional.